src/main/utils/resource.ts

```diff
@@ -44,7 +44,11 @@ export class Resource {
             .download()
             .downloadArtifactToFile(this.sourceUrl, resourcePath);
         if (!(await this.isLocalAndRemoteChecksumMatch(resourcePath))) {
-            throw Error('Local checksum is not match to the remote');
+            if (this._cacheRemoteSha256) {
+                throw Error('Local checksum is not match to the remote');
+            } else {
+                this._logManager.logMessage("Can't get remote checksum header for resource " + this.name, 'WARN');
```

Comment:
Can we also print the actual full remote URL?
Like https://releases.jfrog.io/artifactory/xsc-gen-exe-analyzer-manager-local/v1/[RELEASE]/mac-amd64/analyzerManager.zip

Let's print as much useful information as we can in case of errors.